### DISCUSSION TITLE
Информация под (?)

### DISCUSSION THREAD

**Discussion by kaavain** [cdek-it/widget#26] [Ideas]
Левый-нижний угол, кнопка (?) открывает окно с телефоном и адресом саппорта СДЕКА. Два соображения:
1. Сделайте отображение там текущей версии виджета, или я просто не знаю как ее посмотреть? Эта информация полезна - знать обновился ли он или наблюдаемые глюки еще от старого.
2. Виджет находится на МОЕМ сайте, с большой долей вероятностью ВАШЕМУ саппорту будут задавать вопросы из серии "а есть ли в наличии" или "когда привезете". Это не логично. Надо добавить туда поле (с акцентом) из настроек виджета "телефон/адрес поддержки магазина". Контакты саппорта СДЕКа можно убрать вовсе, оставив только копирайт со ссылкой.

**Comment by vermorag** [comment]
1. Интернет магазин самостоятельно устанавливает виджет на сайт и самостоятельно выбирает версию установки, возможности отобразить версии только для владельца ИМ на карте возможности нет. При включении режима отладки в виджете версии 4.0+ он сообщает свою актуальную версию в логи отладки при инициализации.
2. Когда клиент выбирает способ доставки СДЭК и смотрит подробности доставки при помощи карты - указаны контакты именно транспортной компании, чтобы клиент мог задать вопросы, связанные с выбором пункта получения или информации об условиях доставки. Поскольку аналогичных запросов не возникало, реализовано не будет.
